tst/EnergyPlus/unit/OutputReportTabular.unit.cc

```diff
@@ -311,6 +314,153 @@ TEST( OutputReportTabularTest, GetColumnUsingTabs )
+
+	// refrigLatentSeq.allocate( TotDesDays + TotRunDesPersDays, NumOfTimeStepInHour * 24, NumOfZones );
+	EXPECT_EQ( refrigLatentSeq.size(), 1920u );
+
+	// waterUseInstantSeq.allocate( TotDesDays + TotRunDesPersDays, NumOfTimeStepInHour * 24, NumOfZones );
+	EXPECT_EQ( waterUseInstantSeq.size(), 1920u );
+
+	// waterUseLatentSeq.allocate( TotDesDays + TotRunDesPersDays, NumOfTimeStepInHour * 24, NumOfZones );
+	EXPECT_EQ( waterUseLatentSeq.size(), 1920u );
+
+	// hvacLossInstantSeq.allocate( TotDesDays + TotRunDesPersDays, NumOfTimeStepInHour * 24, NumOfZones );
+	EXPECT_EQ( hvacLossInstantSeq.size(), 1920u );
+
+	// hvacLossRadSeq.allocate( TotDesDays + TotRunDesPersDays, NumOfTimeStepInHour * 24, NumOfZones );
+	EXPECT_EQ( hvacLossRadSeq.size(), 1920u );
+
+	// hvacLossDelaySeq.allocate( TotDesDays + TotRunDesPersDays, NumOfTimeStepInHour * 24, NumOfZones );
+	EXPECT_EQ( hvacLossDelaySeq.size(), 1920u );
+
+	// powerGenInstantSeq.allocate( TotDesDays + TotRunDesPersDays, NumOfTimeStepInHour * 24, NumOfZones );
+	EXPECT_EQ( powerGenInstantSeq.size(), 1920u );
+
+	// powerGenRadSeq.allocate( TotDesDays + TotRunDesPersDays, NumOfTimeStepInHour * 24, NumOfZones );
+	EXPECT_EQ( powerGenRadSeq.size(), 1920u );
+
+	// powerGenDelaySeq.allocate( TotDesDays + TotRunDesPersDays, NumOfTimeStepInHour * 24, NumOfZones );
+	EXPECT_EQ( powerGenDelaySeq.size(), 1920u );
+
+	// infilInstantSeq.allocate( TotDesDays + TotRunDesPersDays, NumOfTimeStepInHour * 24, NumOfZones );
+	EXPECT_EQ( infilInstantSeq.size(), 1920u );
+
+	// infilLatentSeq.allocate( TotDesDays + TotRunDesPersDays, NumOfTimeStepInHour * 24, NumOfZones );
+	EXPECT_EQ( infilLatentSeq.size(), 1920u );
+
+	// zoneVentInstantSeq.allocate( TotDesDays + TotRunDesPersDays, NumOfTimeStepInHour * 24, NumOfZones );
+	EXPECT_EQ( zoneVentInstantSeq.size(), 1920u );
+
+	// zoneVentLatentSeq.allocate( TotDesDays + TotRunDesPersDays, NumOfTimeStepInHour * 24, NumOfZones );
+	EXPECT_EQ( zoneVentLatentSeq.size(), 1920u );
+
+	// interZoneMixInstantSeq.allocate( TotDesDays + TotRunDesPersDays, NumOfTimeStepInHour * 24, NumOfZones );
+	EXPECT_EQ( interZoneMixInstantSeq.size(), 1920u );
+
+	// interZoneMixLatentSeq.allocate( TotDesDays + TotRunDesPersDays, NumOfTimeStepInHour * 24, NumOfZones );
+	EXPECT_EQ( interZoneMixLatentSeq.size(), 1920u );
+
+	// feneCondInstantSeq.allocate( TotDesDays + TotRunDesPersDays, NumOfTimeStepInHour * 24, NumOfZones );
+	EXPECT_EQ( feneCondInstantSeq.size(), 1920u );
+
+	// feneSolarRadSeq.allocate( TotDesDays + TotRunDesPersDays, NumOfTimeStepInHour * 24, TotSurfaces );
+	EXPECT_EQ( feneSolarRadSeq.size(), 3360u );
+
+	// feneSolarDelaySeq.allocate( TotDesDays + TotRunDesPersDays, NumOfTimeStepInHour * 24, NumOfZones );
+	EXPECT_EQ( feneSolarDelaySeq.size(), 1920u );
+
+	// surfDelaySeq.allocate( TotDesDays + TotRunDesPersDays, NumOfTimeStepInHour * 24, TotSurfaces );
+	EXPECT_EQ( surfDelaySeq.size(), 3360u );
+
+	DeallocateLoadComponentArrays();
```

Comment:
@JasonGlazer Is there a reason this deallocation call isn't called from within the `clear_state()` call in `OutputReportTabular.cc`?  If it did, this line could be removed as since EnergyPlusFixture calls the `clear_state` for `OutputReportTabular` [here](https://github.com/NREL/EnergyPlus/blob/Fix-allocation-for-feneSolarDelaySeq/tst/EnergyPlus/unit/Fixtures/EnergyPlusFixture.cc#L334).